src/extension/BatchMintMetadata.sol

```diff
@@ -20,11 +20,11 @@ contract BatchMintMetadata {
   /// @dev Metadata frozen
   error BatchMintMetadataFrozen(uint256 batchId);
 
-  /// @dev Largest tokenId of each batch of tokens with the same baseURI + 1 {ex: batchId 100 at position 0 includes tokens 0-99}
-  uint256[] private batchIds;
+  /// @dev Largest tokenId of each batch of tokens with the same baseURIs + 1 {ex: batchId 100 at position 0 includes tokens 0-99}
+  uint256[] internal batchIds;
```

Comment:
what was this changed to `internal`?

Reply:
to allow the `initLazyMint` function to access it to synchronize the tokenId and the `nextTokenIdToLazyMint`. 

Check this function [here](https://github.com/holographxyz/holograph-protocol/blob/1e427a510412e23a3c5fb179ea301fc34e59e3be/src/token/CustomERC721.sol#L181)